docs/general/rendering.md

```diff
@@ -113,8 +113,25 @@ PR please :)
 
 #### root
 
-PR please :)
+By default the Json Adapter `root` will follow snake case format, like so:
```

Comment:
Maybe just a note at the bottom of the section `Note: the Attributes adapter (default) does not include a resource root.`

No need to mention snake case here, since it's really key-specific, rather than root, IMO.

> The resource root is derived from the class name of the resource being serialized.
> e.g. `UserPostSerializer.new(UserPost.new)` will be serialized with the root `user_post` or `user_posts` according the adapter collection pluralization rules.

I'm not sure about how to write that part, but should be a link, maybe to https://github.com/rails-api/active_model_serializers/blob/fce856fc83771a42cce0dc9994ef449ebe9943c0/docs/general/configuration_options.md#json-api and add there that keys are JSON (always singular/plural) or JSON API(same as JSON unless configured)